classes/local/manager.php

```diff
@@ -134,6 +134,29 @@ public static function get_fs_list() {
         return $found;
     }
 
+    /**
+     * Returns the list of installed and available filesystems.
+     *
+     * @return array
+     */
+    public static function get_available_fs_list() {
+        $result = [];
+        $filesystems = self::get_fs_list();
+        foreach ($filesystems as $filesystem) {
+            if ($filesystem == 'Please, select') {
```

Comment:
will it work with translated strings?

Reply:
oh, it's hard-coded errrr

Reply:
Yep. But good catch. Can be swapped to lang string

Reply:
This is done